climpred/tests/conftest.py

```diff
@@ -237,6 +237,31 @@ def observations_da_1d(observations_ds_1d):
     return observations_ds_1d['SST']
 
 
+@pytest.fixture
+def hindcast_recon_3d(hind_ds_initialized_3d, reconstruction_ds_3d):
+    """HindcastEnsemble initialized with `initialized`, `uninitialzed` and `obs`."""
```

Comment:
`"""HindcastEnsemble with initialized and reconstruction (observations) with `nlat` and `nlon` spatial dimensions."""`